Bottom: 7c27ab720144ababa6558f1eec71d4d4ef5092f2
Top:    70c9572a85411ade030b375cbf7fd907c067a34f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-07 20:38:02 +0000

Refresh of instrument-polygon-slowness

---

diff --git a/src/thermal.c b/src/thermal.c
index 77350b9..83907df 100644
--- a/src/thermal.c
+++ b/src/thermal.c
@@ -470,9 +470,6 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
         return NULL;
       pa->f = arc;
       arc->b = pa;
-
-      arc->f = pa;
-#if 0
       a.StartAngle += 90;
       arc = ArcPoly (&a, a.Clearance);
       if (!arc)
@@ -488,7 +485,6 @@ ThermPoly (PCBTypePtr p, PinTypePtr pin, Cardinal laynum)
       arc->b = pa->f->f;
       arc->f = pa;
       pa->b = arc;
-#endif
       return pa;
     }
 }
